.github/workflows/connector_checklist.yml

```
@@ -0,0 +1,23 @@
+name: Add Connector Merge Checklist
+on:
+  pull_request:
+    types: [opened]
+    paths:
+      - "airbyte-integrations/connectors/source-**"
+      - "airbyte-integrations/connectors/destination-**"
+jobs:
+  checklist_job:
+    name: Add Connector Merge Checklist Job
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Checklist
+        uses: wyozi/contextual-qa-checklist-action@master
+        with:
+          comment-header: "### Before Merging a Connector Pull Request \n\n Wow! What a great pull request you have here! 🎉 \n\n To merge this Connector please ensure the following has been done/considered: \n\n"
```

```suggestion
          comment-header: "### Before Merging a Connector Pull Request \n\n Wow! What a great pull request you have here! 🎉 \n\n To merge this PR, ensure the following has been done/considered for each connector added or updated: \n\n"
```
Updated to reflect that there might be changes to multiple connectors in one PR, so the checklist items should be completed for each connector. Not opinionated on the wording